app/src/main/res/values/dimens.xml

```diff
@@ -39,16 +44,29 @@
     <dimen name="normal_lse">6sp</dimen>
     <dimen name="big_lse">12sp</dimen>
 
+    <!--    view radius-->
+    <dimen name="search_view_corner">20dp</dimen>
+<!--    todo-->
+    <dimen name="big_image_radius">30dp</dimen>
+    <dimen name="small_image_radius">20dp</dimen>
+    <dimen name="big_radius">30dp</dimen>
+    <dimen name="medium_radius">20dp</dimen>
```

Comment:
maybe it would be better to leave one dimen for each dp?

Reply:
fixed